api/src/index.js

```diff
@@ -11,21 +11,90 @@ app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+app.get("/my-route", (req, res) => {
+    res.send("Hi friend");
+  });
+
+app.get("/", async (req, res) => {
+    try {
+      res.json({ message: 'API is working!' });  
+    } catch (error) {
+      res.status(500).json({ error: 'Server error' });
+    }
+  });
+
+
+// Route for future meals
+app.get('/future-meals', async (req, res) => {
+  try {
+    const meals = await knex.raw("SELECT * FROM Meal WHERE `when` > NOW()");
```

Comment:
Nice! you think you could also find a way to use the `knex` ORM instead of using `raw` SQL?
see here: https://knexjs.org/guide/query-builder.html

Reply:
Please could you check again